docs/getting-started/integration.md

```diff
@@ -84,3 +84,18 @@ require(['moment'], function() {
     });
 });
 ```
+
+## Content Security Policy
+
+By default, Chart.js injects CSS directly into the DOM. For webpages secured using [`Content-Security-Policy` (CSP)](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy), this requires to allow `style-src 'unsafe-inline'`. For stricter CSP environments, where only `style-src 'self'` is allowed, the following CSS file needs to be manually added to your webpage:
```

Comment:
In consistency with https://github.com/chartjs/Chart.js/pull/6048#discussion_r254536755

`Content-Security-Policy` (CSP) -> `Content Security Policy` (CSP)

or ?

`Content-Security-Policy` (CSP) -> a `Content-Security-Policy` (CSP) response header

Reply:
Should the link be: https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP
If linking to the header, hyphens should be kept. But if linking to consept, hyphens can be omitted.

Reply:
Good point, I changed the URL to link to the concept and removed hyphens.